src/app/public/modules/data-manager/models/data-view-state-options.ts

```diff
@@ -3,6 +3,10 @@ export interface SkyDataViewStateOptions {
    * The ID of this view.
    */
   viewId: string;
+  /**
+   * The IDs of the olumns able to be displayed for column-based views.
```

Comment:
Typo 'olumns'.

Reply:
Also, I'm tripping up on the word "available". Is that any different than just a list of all the column ids? Can this just be `columnIds` ?

Reply:
I'm fine with just doing `columnIds`. I've updated and fixed the typo.